src/main/scala/mesosphere/marathon/core/event/Events.scala

```diff
@@ -270,3 +270,15 @@ case class MesosFrameworkMessageEvent(
     message: Array[Byte],
     eventType: String = "framework_message_event",
     timestamp: String = Timestamp.now().toString) extends MarathonEvent
+
+/** Event indicating the goal of an instance has changed. For status changes, see InstanceChanged */
+case class InstanceGoalChanged(
+    id: Instance.Id,
+    goal: Goal,
+    condition: Condition,
+    runSpecId: PathId,
```

Comment:
Nit: The instance includes the run spec id. It's is not required for the constructor.

Reply:
✅

Reply:
I don't see this issue be resolved. What am I missing?